addons/website/views/snippets/snippets.xml

```diff
@@ -1150,49 +1150,59 @@
             <t t-if="request.session.get('geoip', {}).get('country_code')">
                 <t t-call="website.snippet_options_conditional_visibility">
                     <t t-set="option_name">⌙ Country</t>
-                    <t t-set="attribute_rule">visibilityValueCountryRule</t>
-                    <t t-set="save_attribute">visibilityValueCountry</t>
-                    <t t-set="attribute_name">data-country</t>
-                    <t t-set="model">res.country</t>
-                    <t t-set="call_with">code</t>
-                    <t t-set="data_fields">[&quot;code&quot;]</t>
+                    <t t-translation="off">
+                        <t t-set="attribute_rule">visibilityValueCountryRule</t>
+                        <t t-set="save_attribute">visibilityValueCountry</t>
+                        <t t-set="attribute_name">data-country</t>
+                        <t t-set="model">res.country</t>
+                        <t t-set="call_with">code</t>
+                        <t t-set="data_fields">[&quot;code&quot;]</t>
+                    </t>
```

Comment:
Scary I did not catch that when merging it, it's probably one of the most frequent error I warn about during reviews :grimacing: However use `t-valuef` for each value instead of the translation=off wrapping like RDE asked here: https://github.com/odoo/odoo/pull/75910/commits/a862a2563765979a42ee78e6d29ea26a4f4a47b0#r715432511 :wink: